src/stories/Quiz/Component/index.stories.tsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-console, react/jsx-props-no-spreading */
```

Comment:
What's the difference between this file and `src/stories/Quiz/Component/index.stories.tsx`?

Reply:
That's a a bad merge. We removed this on main it should be removed here as well